### DISCUSSION TITLE
Django + Vite

### DISCUSSION THREAD

**Discussion by umitcan07** [bounswe/bounswe2024group11#286] [General]
Since we will have to deploy our frontend and backend within a single docker container.
Here is a good library we can use:[ Django + Vite](https://github.com/MrBin99/django-vite)

> Vite is our frontend manager which does the heavy lifting such as running a development server with HMR, building the project etc.

**Comment by Meminseeker** [comment]
We do not run our frontend and backend in the same container, which is very risky. We run them in separate containers, but of course they're defined in the same docker-compose file.

**Reply by Meminseeker** [reply]
On top of that, our database will also run in a separate container as well
